### PR TITLE
EventProbe: capture destination tty device, capture termios flags for tty devices

### PR DESCRIPTION
**Pull request by mmat11**

### CODE REVIEW

File: GPL/Events/EbpfEventProto.h [hunk 1]
```diff
@@ -143,7 +143,10 @@ struct ebpf_process_tty_write_event {
     char tty_out[TTY_OUT_MAX];
     uint64_t tty_out_len;
     uint64_t tty_out_truncated;
+    // Controlling TTY.
     struct ebpf_tty_dev ctty;
+    // Writing TTY.
+    struct ebpf_tty_dev wtty;
```
Comment (lrishi) [inline GPL/Events/EbpfEventProto.h:149]: May be we should call it output destination/target tty

Comment (mmat11) [reply GPL/Events/EbpfEventProto.h]: I called it `tty` and changed the comment above

**Review by lrishi — APPROVED** [review]
lgtm 👍🏽

File: GPL/Events/Process/Probe.bpf.c [hunk 2]
```diff
@@ -250,7 +250,11 @@ static int tty_write__enter(const char *buf, ssize_t count, struct file *f)
         goto out;
 
     struct tty_file_private *tfp = (struct tty_file_private *)BPF_CORE_READ(f, private_data);
-    struct winsize winsize       = BPF_CORE_READ(tfp, tty, winsize);
+    struct tty_struct *tty       = BPF_CORE_READ(tfp, tty);
+    if (BPF_CORE_READ(tty, driver, type) == TTY_DRIVER_TYPE_PTY &&
```
Comment (nicholasberlin) [inline GPL/Events/Process/Probe.bpf.c:254]: Please add a comment about this if block.

**Review by nicholasberlin — APPROVED** [review]

File: GPL/Events/Process/Probe.bpf.c (hunk 2 above)
Comment (mmat11) [reply GPL/Events/Process/Probe.bpf.c:260]: done

**Pull request merged**